kubernetes-client/src/main/java/io/fabric8/kubernetes/client/extended/run/RunConfig.java

```diff
@@ -70,7 +70,7 @@
   private int port;
 
   @Buildable(builderPackage = "io.fabric8.kubernetes.api.builder")
-  public GeneratorRunConfig(String name, String image, String imagePullPolicy, String restartPolicy, String serviceAccount, Map<String, String> labels, Map<String, String> env, Map<String, Quantity> limits, Map<String, Quantity> requests, int replicas, int port) {
+  public RunConfig(String name, String image, String imagePullPolicy, String restartPolicy, String serviceAccount, Map<String, String> labels, Map<String, String> env, Map<String, Quantity> limits, Map<String, Quantity> requests, int replicas, int port) { // NOSONAR
```

Comment:
Did you forget to add this NO SONAR comment?

Reply:
remove*

Reply:
It's on purpose. It removes a Sonar reported code smell (constructor with too many args).
I will replace with `@SupressWarnings` squid annotation. I guess today I woke up with a JavaScript mindset 😅

Reply:
hehe. np